src/index.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import ReactDOM from "react-dom";
+
+import { GlobalStyles } from "./GlobalStyles";
+
+import App from "./App";
+
+ReactDOM.render(
+  <React.StrictMode>
+    <GlobalStyles />
+    <App />
+  </React.StrictMode>,
+  document.querySelector("#root")
```

Comment:
`querySelector()` is drastically slower than more conventional `getElementById()`. It won't, probably, hurt performance of the entire app in your particular case (since used only once), however, the latter looks much more comprehensive, while the former is better suited for more sophisticated queries.